rust/arrow/src/datatypes.rs

```diff
@@ -195,6 +195,13 @@ make_type!(
     0i64
 );
 make_type!(Date32Type, i32, DataType::Date32(DateUnit::Day), 32, 0i32);
+make_type!(
```

Comment:
The spec only provides for `Date32` to have a day dateunit (http://arrow.apache.org/docs/format/Metadata.html#date). Parquet dates I believe are only represented in 32-bit number of days (https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#date)

Reply:
1. According to arrow spec, date with milliseconds should be 64 bit?
2. According to parquet, there is no date type with milliseconds units?

So I think this may be unnecessary.